### ISSUE TITLE
When two devices connected and send/receive text, both device showing same name instead of showing different name according to devices identity in Android .

### ISSUE THREAD

**Issue by mbhuiyanst** [VNAPNIC/flutter_nearby_connections#26]
Is this same case for all who already explore with this example code for this pluggin

**Comment by abhay-s-rawat** [comment]
yes its same for example plugin also. I am also facing this issue in 2022. Its been a year for this ?

**Comment by mbhuiyanst** [comment]
I already resolved this issue.

Sent from Yahoo Mail on Android 
 
  On Fri, Apr 29, 2022 at 12:40, ***@***.***> wrote:   


yes its same for example plugin also. I am also facing this issue in 2022. Its been a year for this ?

**Comment by abhay-s-rawat** [comment]
Actually the problem was here:
In browse mode on click of connect button, I was passing the device name of what I was getting instead of mine. Sorry for the confusion.
```
nearbyService.invitePeer(
          deviceID: device.deviceId,
          deviceName:device.deviceName, //We have to pass our device name here deviceName: 'my devce name'
        );

**Comment by natgross** [comment]
> I was passing the device name of what I was getting instead of mine
Ok. So how do I get my device? (I can discover other devices, but not mine?)
